src/main/java/seedu/address/model/widget/WidgetObject.java

```diff
@@ -12,37 +12,54 @@
  *     <li>Text 1</li>
```

Comment:
"standardise" should be "standardised"

Reply:
Thats weird the preview does not show the change.